### ISSUE TITLE
Cast from Array(Redis::RedisValue) to (String | Nil) failed when using brpoplpush

### ISSUE THREAD

**Issue by parruda** [stefanwille/crystal-redis#68]
Forgive me if that's just because I am just learning Crystal, but I am stuck on this.
I am trying to pop something from a queue with:

```
message = redis.brpoplpush("queue_name", "process_queue_name", 15)
```

The docs say:

> Return value: String, the element being popped from source and pushed to destination. If timeout is reached, nil is returned

Once timeout is reached, I am getting this error:

```
Exception: cast from Array(Redis::RedisValue) to (String | Nil) failed, at /dev/lib/redis/src/redis/command_execution/value_oriented.cr:37:9:37 (TypeCastError)
```

What am I doing wrong?

**Comment by parruda** [comment]
It works.
Thanks a lot @maiha !

**Comment by stefanwille** [comment]
I have merged @maiha's PR and published release 2.1.1. Please try!

**Comment by parruda** [comment]
It works. Thanks!
